providers/oidc.go

```diff
@@ -5,14 +5,16 @@ import (
 	"encoding/json"
 	"fmt"
 	"reflect"
+	"strconv"
 	"strings"
 	"time"
 
+	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/logger"
```

Comment:
Let's clean up these imports and get it back together and sorted. I suspect an overeager IDE is to blame 😄

Reply:
Yep